src/util.js

```diff
@@ -118,3 +118,36 @@ export const getRGBAColor = (baseColor, alpha) => {
 
   return `rgba(${r},${g},${b},${alpha})`;
 };
+
+/**
+ * @function
+ * @name truncateString
+ * @description truncates string
+ * @param {string} str  string to truncate
+ * @param num
+ * @param {number} length a string  will be truncated to
+ * @returns {string} truncated string
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+export const truncateString = (str, num) => {
+  // If the length of str is less than or equal to num
+  // just return str--don't truncate it.
+  if (str.length <= num) {
+    return str;
+  }
+  // Return str truncated with '...' concatenated to the end of str.
+  return `${str.slice(0, num)}...`;
+};
+
+/**
+ * @function
+ * @name isString
+ * @description checks if a variable is a string
+ * @param {string} data  string to truncate
+ * @returns {boolean} truthy value based on comparison
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+export const isString = data =>
+  typeof data === 'string' || data instanceof String;
```

Comment:
@Mloweedgar Hello, Can we use lodash for utility function like this to avoid reinventing the wheel?

Reply:
sure we can